# This message is for storing data from the BME680 chip https://shop.pimoroni.com/products/bme680-breakout
# temperature is provided in celsius
# humidity is in percent of how much of the moisture capacity of the air is used
# pressure is provided in hectopascal (hPa, 1 hPa = 100 Pa)
# air quality is provided in Ohms
Header header
float32 temperature
float32 humidity
float32 pressure
float32 air_quality